pallets/xcmp-queue/src/lib.rs

```diff
@@ -130,6 +130,38 @@ pub mod pallet {
 			Self::deposit_event(Event::OverweightServiced(index, used));
 			Ok(Some(used.saturating_add(1_000_000)).into())
 		}
+
+		/// Overwrites QueueConfig storage item.
+		///
+		/// - `origin`: Must pass `Root`.
+		/// - other parameters: refer to `QueueConfigData` Struct items
+		///
+		/// - Events:
+		/// - `UpdatedConfig`: On Success.
+		#[pallet::weight(10_000_000 as Weight + T::DbWeight::get().writes(1))]
+		pub fn update_config(
+			origin: OriginFor<T>,
+			suspend_threshold: u32,
+			drop_threshold: u32,
+			resume_threshold: u32,
+			threshold_weight: Weight,
+			weight_restrict_decay: Weight,
+			xcmp_max_individual_weight: Weight,
```

Comment:
Hmm, these should be either all `Option`s or we just create one call per value to set. Like it is done for `HostConfiguration` in Polkadot